package.json

```diff
@@ -15,31 +15,26 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "peerDependencies": {
-    "apollo-cache-inmemory": "^1.1.5",
-    "apollo-client": "^2.2.0",
-    "apollo-link": "^1.0.7",
-    "apollo-link-http": "^1.3.2",
-    "bs-platform": "^2.1.0",
-    "graphql-tag": "^2.6.1",
+    "apollo-cache-inmemory": "^1.2.1",
+    "apollo-client": "^2.3.1",
+    "apollo-link": "^1.2.2",
+    "apollo-link-http": "^1.5.4",
+    "bs-platform": "^3.1.3",
+    "graphql-tag": "^2.9.2",
     "graphql_ppx": "^0.2.0",
```

Comment:
You can use the latest `0.2.3`. The bug has been fixed https://github.com/mhallin/graphql_ppx/releases/tag/0.2.3